hal/linux/package/agent/agent.mk

```diff
@@ -12,7 +12,14 @@ define AGENT_BUILD_CMDS
 endef
 
 define AGENT_INSTALL_TARGET_CMDS
-	cp $(@D)/build/cocos-agent $(TARGET_DIR)/bin
+	$(INSTALL) -D -m 0755 $(@D)/build/cocos-agent $(TARGET_DIR)/bin
+	mkdir -p $(TARGET_DIR)/var/log/cocos
 endef
 
-$(eval $(golang-package))
\ No newline at end of file
+define AGENT_INSTALL_INIT_SYSTEMD
+	$(INSTALL) -D -m 0644 $(@D)/init/systemd/cocos-agent.service $(TARGET_DIR)/usr/lib/systemd/system/cocos-agent.service
+	$(INSTALL) -D -m 0644 $(@D)/init/systemd/eth0.network $(TARGET_DIR)/etc/systemd/network
+	$(INSTALL) -D -m 0644 $(@D)/init/systemd/00-network.link $(TARGET_DIR)/etc/systemd/network
```

Comment:
I have changed the name of the .link file.

There are a couple of ways for a network interface to get its name:

1. By kernel - this can be unpredictable if there are multiple network interfaces because the kernel does device probing in random order and thus can give different names to network interfaces during reboots. For example, the first ethernet device detected gets the name `eth0`, the second `eth1`, and so on. We have only one interface, so it will be named `eth0`.
2. By `systemd` - the `systemd` gives predictable names and provides them based on their location in the system. 
3. By user - using ip command or some other way.

The name of the file starts with numeric numbers in order for it to be processed by the system before the default network configuration file. From the documentation:
```The link file contains a [Match] section, which determines if a given link file may be applied to a given device, as well as a [Link] section specifying how the device should be configured. The first (in lexical order) of the link files that matches a given device is applied. Note that a default file 99-default.link is shipped by the system. Any user-supplied .link should hence have a lexically earlier name to be considered at all.```